### DISCUSSION TITLE
Add a checkbox that allows you to choose to have a sound notification at the end of the training

### DISCUSSION THREAD

**Discussion by Blu3train** [buliasz/tesstrain-windows-gui#17] [Ideas]
Usually when I start the training, I turn off the monitor and leave the computer to process.
In the script that I had done before, at the end of the training I put a sound in such a way that I realize that it has finished.

Would it be possible to add a checkbox that allows you to choose to have a sound notification at the end of the training?
Thanks and congratulations for the excellent work.

**Comment by buliasz** [comment]
Hi Blu3train,
I'm not sure if we need configuration for this. I can put standard system beep at the end of training unconditionally.
But if you see a reason to make it configurable please let me know?

**Comment by Blu3train** [comment]
That's fine. 
The checkbox was just because I thought maybe someone might bother my request, but for me having the standard system beep unconditionally is fine.

**Reply by buliasz** [reply]
Done in the new 4.3 version.

**Comment by Blu3train** [comment]
In the _tesstrain.ahki file I added the SoundBeep command in the StartTraining() function.
I tried and at the end of the training it makes a very audible beep.

```
StartTraining() {
	if (!FileExist(LAST_CHECKPOINT)) {
		if (!TrainAndProduceCheckpoints()) {
			StatusUpdate("Training error")
			return false
		}
	}
	SoundBeep 800, 300
	Checkpoint2Traineddata(LAST_CHECKPOINT, DATA_DIR "\" MODEL_NAME ".traineddata", false)
	StatusUpdate("Training finished")
	return true
}

```

**Comment by buliasz** [comment]
Thanks! I've merged your change.
